uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -198,6 +201,10 @@ define([
           canSubmit = canSubmit && !_.isEqual(initialLeaveRequestAttributes, this.request.attributes());
           // user has added a comment
           canSubmit = canSubmit || this.request.comments.length !== initialCommentsLength;
+          // user has added file Attachment
+          canSubmit = canSubmit || (this.request.files.length + this.request.fileUploader.queue.length) !== initialFilesLength;
+          // did user remove file
+          canSubmit = canSubmit || this.userRemovedFile;
```

Comment:
Why is it important, in order to understand if someone can submit or not the request, to know whether a user has removed or not a file?